README.md

```diff
@@ -1,2 +1,16 @@
-# myWebsite
-My personal webpage
+# title(){return this.options.title||this.options.name}
```

Comment:
Write a more descriptive README.md
Learn in less than 10 minutes how to write a .md file using markdown.

Reply:
I've done my research about linters and simply put they are tools that analyze source code and search for common issues, errors, stylistic errors, and suspicious constructs.